reliability/README.md

```diff
@@ -1,81 +1,205 @@
+## Configuration
+Example [config](https://github.com/openshift/svt/blob/master/reliability/config/example_reliability.yaml). Below sections explains each part of the configuration file.
+
+### Files needed
+
+If you installed your cluster with [Flexy-install](https://mastern-jenkins-csb-openshift-qe.apps.ocp4.prod.psi.redhat.com/job/ocp-common/job/Flexy-install/), download kubeconfig, users.spec and kubeadmin-password files. If you installed cluster in other way, prepare the above files accordingly.
+
+Replace kubeconfig kubeadmin_file and user_file in your config file.
+```yaml
+reliability:
+  kubeconfig: <path_to_kubeconfig>
+  users:
+    - kubeadmin_file: <path_to_kubeadmin-password>
+    - user_file: <path_to_users.spec>
+```
+
+### Tasks timeSubstitutions
+[Tasks](#Tasks) are classified as minutely, hourly, weekly or monthly.  To each of these intervals, a real time interval is assigned.  This is done to allow assigning activities with a natural spacing while allowing for faster simulation of the activity. For example, in the following config, `day` tasks will be exectued in every 2 minutes.
 ```yaml
 reliability:
   timeSubstitutions:
-    minute: 20s
-    hour: 1m
+    minute: 10s
+    hour: 30s
     day: 2m
     week: 3m
     month: 4m
+```
+
+### Limits
+Limits limits the max project can be created during the reliability test. The number depends on the size of the cluster. Some recomendations can be found in the example [config](https://github.com/openshift/svt/blob/master/reliability/config/example_reliability.yaml)
+```yaml
+reliability:
   limits:
-    maxProjects: 20
+    # total number of projects to create
+    # for 3 nodes m5.xlarge cluster, 25 to 30 is recomended
+    # for 5 nodes m5.xlarge cluster, 60 is recomended
+    maxProjects: 25
     sleepTime: 10
+```
 
-  appTemplates:
-    - template: cakephp-mysql-example
-    - template: nodejs-mongodb-example
-    - template: django-psql-example
-    - template: rails-postgresql-example
-    - template: dancer-mysql-example
-  users:
-    - id: redhat
-      pw: redhat
-    - id: test
-      pw: test
+### Cerberus Integration
+Reliablity can integrate with [Cerberus](https://github.com/cloud-bulldozer/cerberus) to check the healthy of the cluster and take action accordingly.
+
+The below configuration enables the Cerberus integration `cerberus_enable: True`, and provided the Cerberus api `cerberus_api: "http://0.0.0.0:8080"` where Reliability test can get the [Cerberus status and history](https://github.com/cloud-bulldozer/cerberus#metrics-api) from. The `cerberus_fail_action` configures how Reliability test acts when Cerberus status is False.
+
+`pause`: When Cerberus status is 'False', pause Reliability test until Cerberus status is changed to 'True'."
+
+`halt`: When Cerberus status is 'False', halt Reliablity test.
```

Comment:
It's not super clear here what `halt` does. Does it end the reliability test?

Reply:
Yes, just like touching a halt file. I can make it clearer.